mods/cnc/chrome/ingame.yaml

```diff
@@ -39,6 +39,11 @@ Container@INGAME_ROOT:
 				Container@PLAYER_ROOT:
 		Container@MENU_ROOT:
 		TooltipContainer@TOOLTIP_CONTAINER:
+		Label@MISSION_TEXT:
+			X: 500
+			Y: 22
+			Height: 25
+			Font: Bold
```

Comment:
Can you center the label across the screen? Something like the snippet below might work.
````
X: (WINDOW_RIGHT/2) - 256
Width: 512
TextAlign: Center
```

Not sure about the 256/512 bit. Might still collide with the radar.